BambooTracker/io/export_io.cpp

```diff
@@ -107,14 +107,36 @@ void writeVgm(BinaryContainer& container, int target, const std::vector<uint8_t>
 		break;
 	}
 
+	// 0xbc: Extra header offset
+	*reinterpret_cast<uint32_t*>(header + 0xbc) = mix ? (0x100 - 0xbc) : 0;
+
 	container.appendArray(header, 0x100);
 
+	// Extra header
+	if (mix) {
+		// Extra header size
+		container.appendUint32(17);
```

Comment:
This should actually be only "12".

I may need to clarify the specification: The "extra header size" covers only the pointer list itself and not the additional data that is pointed to.